test/System/MemoryInputStream.cpp

```diff
@@ -0,0 +1,30 @@
+#include <SFML/System/MemoryInputStream.hpp>
+#include "SystemUtil.hpp"
+
+#include <doctest.h>
+
+TEST_CASE("sf::MemoryInputStream class - [system]")
+{
+	SUBCASE("Empty stream")
```

Comment:
`getSize()` should also be tested in this subcase.
Also, please indent with spaces instead of tabs.

Reply:
Yeah didn't notice it was set to tabs, I'll add a `getSize()` check and format it :)